client/cypress/integration/query/parameter_spec.js

```diff
@@ -21,25 +37,35 @@ describe('Parameter', () => {
         .then(({ id }) => cy.visit(`/queries/${id}`));
     });
 
-    it('updates the results after clicking in Apply or pressing enter', () => {
+    it('updates the results after clicking Apply', () => {
       cy.getByTestId('ParameterName-test-parameter')
         .find('input')
         .type('Redash');
 
-      cy.getByTestId('ParameterName-test-parameter')
-        .contains('Apply')
+      cy.getByTestId('ParameterApplyButton')
         .click();
 
       cy.getByTestId('DynamicTable')
         .should('contain', 'Redash');
 
       cy.getByTestId('ParameterName-test-parameter')
         .find('input')
-        .type('{selectall}New value{enter}');
+        .type('{selectall}New value');
+
+      cy.getByTestId('ParameterApplyButton')
+        .click();
```

Comment:
Those tests had two parts: One clicking in the apply an another pressing enter. If you want you can remove this second part.

Alternatively you can test CTRL+Enter, but this one can also be one separate test for all types.

Reply:
Yes! I'll use `{meta}{enter}` 👍

Reply:
There are now 3 separate apply tests:
* applies changes from "Apply Changes" button
* applies changes from "Execute" button
* applies changes from "meta+enter" keyboard shortcut